pom.xml

```diff
@@ -124,7 +185,7 @@
         <confluence.data.version>6.9.0</confluence.data.version>
         <amps.version>8.0.0</amps.version>
         <plugin.testrunner.version>1.1</plugin.testrunner.version>
-        <atlassian.spring.scanner.version>2.1.7</atlassian.spring.scanner.version>
+        <atlassian.spring.scanner.version>1.2.13</atlassian.spring.scanner.version>
```

Comment:
Please review this in context of https://ecosystem.atlassian.net/browse/SCANNER-2170